tests/fixtures/vcr/test_account_information.yaml

```diff
@@ -31,4 +31,36 @@ interactions:
       - '117635'
     http_version: HTTP/1.1
     status_code: 200
```

Comment:
how did all the fixtures get new requests?

Reply:
bumped version from merging in main